kolibri/core/tasks/api.py

```diff
@@ -290,7 +341,9 @@ def create(self, request):
 
                 request_data = validator_result
 
-            job_id = registered_job.enqueue(**request_data)
+            job_id = registered_job.enqueue(
+                facility_id=request.user.facility_id, **request_data
```

Comment:
This is not working when is being executed from the setup wizard. In this case the request is done from an anonymous user that in the previous implementation has permission to execute this endpoint because it's in a non-provisioned device. 
Now we get this error and the wizard can not sync any user:
```
  File "/datos/le/mio/kolibri/kolibri/core/tasks/api.py", line 345, in create
    facility_id=request.user.facility_id, **request_data
AttributeError: 'KolibriAnonymousUser' object has no attribute 'facility_id'
```

Reply:
Looks like I should remove the `[IsAuthenticated]` permission class from the `permission_classes` and use `request.user.get(...)` instead of accessing the attribute directly.